File: aplicaciongrabar/MainActivity.java
package com.example.aplicaciongrabar;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    EditText nombre;
    Button grabar;
    TextView texto;
    EditText apellido;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        nombre=findViewById(R.id.et_nombre);
        grabar=findViewById(R.id.bt_mostrar);
        texto=findViewById(R.id.tv_texto);
        apellido=findViewById(R.id.et_apellido);

        grabar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String name=nombre.getText().toString();
                String surname=apellido.getText().toString();
                texto.setText(name+","+surname);
            }
        });
    }
}

File: primerpasokotlin/MainActivity.kt
package com.example.primerpasokotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var alumno=Alumno()
        btn_saludar.setOnClickListener(View.OnClickListener {
            alumno.nombre=et_nombre.text.toString()
            tv_saludar.text="te llamas "+alumno.nombre
        })
    }
}